### PR TITLE
Mes 5280/candidate declaration page

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

Created pages for candidate declaration for CPC

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

Screenshot 2020-06-09 at 10 32 35

### CODE REVIEW

File: src/pages/page-names.constants.ts [hunk 1]
```diff
@@ -162,6 +162,7 @@ export const CAT_CPC = {
   BACK_TO_OFFICE_PAGE: 'BackToOfficeCatCPCPage',
   COMMUNICATION_PAGE: 'CommunicationCatCPCPage',
   DEBRIEF_PAGE: 'DebriefCatCPCPage',
+  HEALTH_DECLARATION_PAGE: 'HealthDeclarationCatHomeTestPage',
```
Comment (matthew2564) [inline src/pages/page-names.constants.ts]: CatHomeTest?

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.signiturePadding {
```
Comment (matthew2564) [inline src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss]: change spelling to 'signaturePadding'

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.ts [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  licenseProvided$: Observable<boolean>;
+  conductedLanguage$: Observable<string>;
+}
+
+@IonicPage()
+@Component({
+  selector: '.health-declaration-cat-cpc-page',
+  templateUrl: 'health-declaration.cat-cpc.page.html',
+})
+export class HealthDeclarationCatCpcPage extends BasePageComponent {
```
Comment (matthew2564) [inline src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.ts]: other CPC pages have uppercased category name i.e. HealthDeclarationCatCPCPage

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss (hunk 2 above)
Comment (andrewsetterfield) [reply src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss]: will do

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.module.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      HealthDeclarationAnalyticsEffects,
+      HealthDeclarationEffects,
+    ]),
+    HealthDeclarationComponentsModule,
+    ComponentsModule,
+    ReactiveFormsModule,
+    TranslateModule,
+  ],
+})
+export class HealthDeclarationCatCpcPageModule { }
```
Comment (matthew2564) [inline src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.module.ts:28]: uppercase category i.e. HealthDeclarationCatCPCPageModule

File: src/pages/page-names.constants.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/pages/page-names.constants.ts]: will change to CPC

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.module.ts (hunk 4 above)
Comment (andrewsetterfield) [reply src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.module.ts:28]: will update

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.ts (hunk 3 above)
Comment (andrewsetterfield) [reply src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.ts]: will do

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.signaturePadding {
```
Comment (Cullima) [inline src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss]: can we not use came case for css class names. can you rename to cpc-signature-padding

**Review by Cullima — APPROVED** [review]

**Review by matthew2564 — APPROVED** [review]

File: src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss (hunk 5 above)
Comment (andrewsetterfield) [reply src/pages/health-declaration/cat-cpc/health-declaration.cat-cpc.page.scss]: updated

**Pull request merged**